src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -373,7 +373,7 @@ def transform_summary(
 
         # Add a per_organ aggregate to the ElasticSearch request
         es_search.aggs.bucket(
-            'group_by_organ', 'terms', field='contents.cell_suspensions.organ.keyword'
+            'group_by_organ', 'terms', field='contents.cell_suspensions.organ.keyword', size=99999
```

Comment:
Can you make this an attribute in `azul.Config`. You'd have to hunt down every `terms` aggregation but I just did a search for `terms` and it came up with 48 matches half of which are obviously not aggregations.